### ISSUE TITLE
Motion takes a snapshot every 10 minutes but is set for 1 hour

### ISSUE THREAD

**Issue by ve7bul** [Motion-Project/motion#432]
1. version [x.y.z, hash, other]: 
2. installed as a package or compiled from sources [deb, rpm, git, other]: 
3. standalone or part of third party [motion, MotionEyeOS, other]: 
4. video stream source [V4L (card or USB), net cam (mjpeg, rtsp, other), mmal]: 
5. hardware [x86, ARM, other]: 
6. operating system [Linux (which), FreeBSD, other]:

**Comment by ve7bul** [comment]
I have 2 RPi 3 Bs 32GB, SD cards, running latest Raspbian. Both updated and upgraded and Motion installed. Both running same cameras (USB) -something from China. Raspbian installed via NOOBS. Both with Apache2, Samba, and Pagekite. I installed on (1) the Mopower UPS wrapper and shield and then disabled it as I thought it might be the problem.
Both motion.conf files are EXACTLY the same.
RPi (1) has the time between snapshot intervals set to 3600 as does the second one (2). Rpi (1) takes a snapshot every 10 minutes and (2) works as it should taking a snapshot every hour. Anyone have this happen to them or any insight as to what the problem might be as I cannot seem to see anything wrong? I don't know if this is somehow an externally triggered event or how it could be. 
I am lost as to what to check next but I think it might be something simple.
Thanks for having a look.

**Comment by genius3000** [comment]
You missed a few key criteria in skipping the Issue Template:
- Version(s) of Motion running
- How installed (build from source, .deb releases, package manager)

Skipping ahead of that, I would recommend:
1. Ensure the time is accurate on the "wrong" Pi, though this is unlikely the issue if it's a proper 10 minutes each time.
2. Make sure the correct config file is being read. Some packages rely on /etc/motion.conf (or maybe /etc/motion/motion.conf) by default.
3. Enable more verbose/debug logging and see if it leads to any clues as to why it's triggering every 10 minutes.

If that fails, we'll likely need to see the config files and run logs (pastebin, gist, etc).

**Comment by ve7bul** [comment]
Thanks very much for the reply.
Clock and Local parameters set through raspi-config ca-utf-8 and
America/Vanvouver
Motion was installed from the command line: sudo apt-get install motion.
Config file:  This config file was generated by motion 3.2.12+git20140228
The config file is contained in: /etc/motion/motion.conf   If there is
another config file elsewhere I don't know about it, and cannot find it.
If I: snapshot_interval 0  AND: sudo service motion restart OR I:
;snapshot_interval 3600 AND sudo service motion restart it appears to
effectively turn off the snapshot function as I no longer accumulate
snapshots at any interval
Unfortunately I am unfamiliar and not well versed in doing the more verbose
and log file thing as i am not very good with Linux in general. Most of the
things I try are the hard way and I have no one locally I can ask for
input, so everything I have managed to do has been by Google and the
articles/tutorials by others, a LOT of which appear to be incorrect or
outdated.
I still think there MAY be something to do with the Mopower UPS as I
believe it has a RTC and uses the serial interface which you have to
disable during the install. I noticed that Motion COULD want access to the
serial interface for the PTZ functions and I don't know if this is causing
a conflict but in Motion the function appears to be disabled.
Thanks again for the reply

**Comment by tosiara** [comment]
You will have to upload full config of a "not working" motion and provide full log so we can see at which time stamps pictures are being saved

**Comment by Mr-DaveDev** [comment]
@ve7bul You need to upgrade to a recent version before we can assist.   At a minimum 4.0.1 via the deb packages on our [release](https://github.com/Motion-Project/motion/releases) page but preferably to the git master.  Once that is complete, please upload the full config to gist so that we can attempt to replicate.

**Comment by ve7bul** [comment]
Well, you are definitely getting out of my comfort zone here. I am not sure
of how to download and install the 4.0.1 Motion and/or whether I have to
uninstall/purge the Motion I have now. I had a look for the git master and
appeared to get nowhere as I did not find a file that was the obvious thing
to download. I would have thought that the standard install repository
would have contained the latest releases of the software but that is
evidently not the case. Any direction here would be appreciated. Thanks for
getting back to me. BTW: What's a gist?

**Comment by Mr-Dave** [comment]
The following is our [guide](http://htmlpreview.github.io/?https://github.com/Motion-Project/motion/blob/master/motion_guide.html#Install_Build) which includes the instructions.  If you are not comfortable compiling, just stick with the deb packages.  

gist (https://gist.github.com/) allows for pasting of code/items that are large and shouldn't be posted into issues.

**Comment by Mr-Dave** [comment]
Closing for the moment.  We will need the issue replicated via a current version in order to replicate.  

Also, if the instructions included in the guide for installing from a deb / release were not clear, please open separate issue of "Add additional clarity on installing from deb" and include very specific details on what process was followed while trying to upgrade and what items were not clear in the guide instructions.

**Comment by ve7bul** [comment]
Well, you may happen to notice that part of the prerequisites asked for,
and my reply was that I installed Motion through: sudo apt-get motion
which, in the guide is one of the acceptable ways of installing. This also
begs the question as to why it was not the latest version of Motion that
was installed.... Regardless, I will attempt to install from a release deb
package, as per the instructions. Hopefully that will work.

**Comment by ve7bul** [comment]
Well, that did NOT work and performs exactly the same as the original
version of Motion.

**Comment by ve7bul** [comment]
It would appear that running gist from a Windows laptop to a headless Pi to
copy a file to post may be a bit of a problem as gist appears to ask for
the file name to copy etc. and that is not on the Windows laptop.

Closing the issue does not solve the problem.

I can try to move everything to a keyboard and monitor and then try gist.

**Comment by ve7bul** [comment]
Yeah, right.
You might as well delete this thread as I think this is not going to
happen. I tried to copy my .conf and there is no way I can get it to go.
Thanks to everyone who tried to give input. 73.
